docker/lnd/start-lnd.sh

```diff
@@ -52,6 +53,8 @@ RESTLISTEN=$(set_default "$RESTLISTEN" "8080")
 RPCLISTEN=$(set_default "$RPCLISTEN" "10009")
 MONITORLISTEN=$(set_default "$MONITORLISTEN" "8989")
 CHAN_CONFS=$(set_default "$CHAN_CONFS" 3)
+ZMQPUBRAWBLOCK=$(set_default "$ZMQPUBRAWBLOCK" "tcp://localhost:28332")
```

Comment:
Might be less error prone if the `localhost` here is parameterized. This will be the same as the `RPCHOST` right? Just with different ports?